app/src/main/java/com/github/libretube/api/StreamsExtractor.kt

```diff
@@ -58,9 +58,11 @@ object StreamsExtractor {
             category = resp.category,
             views = resp.viewCount,
             likes = resp.likeCount,
-            dislikes = if (PlayerHelper.localRYD)
-                RetrofitInstance.externalApi.getVotes(videoId).dislikes
-            else 0,
+            dislikes = runCatching {
+                if (PlayerHelper.localRYD)
+                    RetrofitInstance.externalApi.getVotes(videoId).dislikes
+                else 0
+            }.getOrElse { 0 },
```

Comment:
```suggestion
            dislikes = if (PlayerHelper.localRYD) runCatching {
                RetrofitInstance.externalApi.getVotes(videoId).dislikes
            }.getOrElse { -1 } else -1,
```
The try catch block doesn't need to be executed when the option is disabled anyways.

Apart from that we might want to return -1 instead of 0 elsewise since 0 could be interpreted as a normal dislike count